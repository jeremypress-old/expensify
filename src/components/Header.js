import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink activeClassName="is-active" exact={true} to="/">Go Home!</NavLink>
        <NavLink activeClassName="is-active" to="/create">add expense</NavLink>
        <NavLink activeClassName="is-active" to="/edit">edit expense</NavLink>
        <NavLink activeClassName="is-active" to="/help">help</NavLink>

    </header>
)

export default Header;
