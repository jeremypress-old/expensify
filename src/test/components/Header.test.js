import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('should render header correctly', () =>{
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find('h1').length).toBe(1);
    // const renderer = new ReactShallowRenderer();
    // renderer.render = (<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})
