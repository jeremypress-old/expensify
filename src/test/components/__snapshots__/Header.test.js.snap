// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Go Home!
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    add expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/edit"
  >
    edit expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    help
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
